src/fastd.c

```diff
@@ -256,11 +266,12 @@ static inline void write_pid(void) {
 		pr_error_errno("can't write PID file: open");
 		goto end;
 	}
+	FILE *f = fdopen(fd, "w");
```

Comment:
The `fopen` manpage says the following:

   Any created files will have mode S_IRUSR | S_IWUSR | S_IRGRP | S_IWGRP | S_IROTH | S_IWOTH (0666), as modified by the process's umask value (see umask(2)).